clients/venice-push-job/src/main/java/com/linkedin/venice/hadoop/DefaultInputDataInfoProvider.java

```diff
@@ -296,6 +283,17 @@ private VeniceVsonFileIterator getVeniceVsonFileIterator(FileSystem fs, Path pat
 
   @Override
   public byte[] trainZstdDictionary() {
+    int collectedNumberOfSamples = pushJobZstdConfig.getCollectedNumberOfSamples();
```

Comment:
I didn't move that because it also emits a metric, and I think we would like to have that metric till we keep that job around. With this change, there is no way to signal the caller if the dict is based on data, or dummy data